bin/GToTree

```diff
@@ -242,6 +242,16 @@ if [ -f "$NCBI_acc_file" ]; then
     fi
 fi
 
+# checking no duplicates labels in label file
+if [ -f "$file_to_genome_id_map" ]; then
+    num_dupes=$(cut -f 2 -d$'\t' "$file_to_genome_id_map" | uniq -d | wc -l)
```

Comment:
Thanks for throwing this in here! 

I only notice 2 nuanced things we need to account for:

1) Before using `uniq` we need to sort things, just cause it won't recognize matches unless they are sequential. 
2) We can't only look at the 2nd column, because they might be identical, but the 3rd column could have a value being appended that would ultimately make the final label unique. For example, the mapping file might look like this:

![image](https://user-images.githubusercontent.com/13923308/78418107-f1a45880-75ed-11ea-9a97-dd745cbfa64b.png)


Where the second column has "Alteromonas macleodii" twice, but the first one will have "SPECIAL" appended to it. 

If you change this line 247 to this, that should cover both :) 

`num_dupes=$(cut -f 2,3 "$file_to_genome_id_map" | tr "\t" "_" | sort | uniq -d | wc -l)`